manifests/config_registry.pp

```diff
@@ -0,0 +1,34 @@
+class conjur::config_registry inherits conjur {
+  
+  registry_key { 'HKLM\Software\CyberArk\Conjur':
+    ensure => present,
+  }
+
+  if $conjur::ssl_certificate {
+    registry_value { 'HKLM\Software\CyberArk\Conjur\SslCertificate':
+      ensure => present,
+      type   => string,
+      data   => $conjur::ssl_certificate,
+    }
+  } else {
+    $cert_file = undef
```

Comment:
Why is this needed if we're storing the cert verbatim in the registry?

BTW, I was thinking, should we use system certificate storage instead?

Reply:
> Why is this needed if we're storing the cert verbatim in the registry?

Ah, thanks. It's not needed.

Reply:
> BTW, I was thinking, should we use system certificate storage instead?

Using the Cert Manager seems like a good idea. My suggestion is that we explore that as a follow on issue in parallel with getting the CI for this one set up. It's hard to tell just how much effort is or isn't required for adding Cert Manager support. 

CC: @jvanderhoof @boazmichaely @garkler, if we have the time to do it, this seems like a worthwhile additional enhancement.

Reply:
@micahlee, I added this to the Detailed Feature Spec as a GH suggestion under a `Future Work` section.